Content.Server/GameObjects/Components/Mobs/Speech/AccentManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Content.Server.Interfaces.Chat;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components.Mobs.Speech
+{
+    public interface IAccentManager
+    {
+        public void Initialize();
+    }
+
+    public class AccentManager : IAccentManager
+    {
+        public void Initialize()
+        {
+            IoCManager.Resolve<IChatManager>().RegisterChatTransform(AccentHandler);
```

Comment:
You can move the two `IoCManager` calls in this class to be dependencies of `AccentManager` and have the first line of its initialize method (or an empty constructor) call `IoCManager.InjectDependencies(this);`